AWSIoTTests/AWSIoTDataManagerTests.swift

```diff
@@ -150,6 +184,19 @@ class AWSIoTDataManagerTests: XCTestCase {
         tokenKeyName = packageConfig["custom_authorizer_token_key_name"]
         tokenSignature = packageConfig["custom_authorizer_token_signature"]
         tokenValue = packageConfig["custom_authorizer_token_value"]
+        
+        customAuthorizerName_customAuthUserPass = packageConfig["custom_authorizer_user_pass_name"]
+        tokenKeyName_customAuthUserPass = packageConfig["custom_authorizer_user_pass_token_key_name"]
+        tokenSignature_customAuthUserPass = packageConfig["custom_authorizer_user_pass_token_signature"]
+        tokenValue_customAuthUserPass = packageConfig["custom_authorizer_user_pass_token_value"]
+
+        let custom_authorizer_user_pass_username = packageConfig["custom_authorizer_user_pass_username"]
+        guard let entireUserName = custom_authorizer_user_pass_username else {
```

Comment:
OOC, why do we pass the entire username in the test config, rather than the username string? Now the test has to know how to split on question mark, which technically shouldn't be something the test code is doing, since we should be exercising (as much as practical) the public API

Reply:
yea.. i was on the fence about this... thanks for calling this out.  I've updated the CDK code to just use "aws-sdk-user" as the user name.